api/envoy/api/v2/endpoint/load_report.proto

```diff
@@ -53,9 +91,8 @@ message UpstreamLocalityStats {
   // Stats for multi-dimensional load balancing.
   repeated EndpointLoadMetricStats load_metric_stats = 5;
 
-  // [#not-implemented-hide:] The priority of the endpoint group these metrics
-  // were collected from.
-  uint32 priority = 6;
+  // Opaque metadata extending the Stats information.
```

Comment:
Can you comment on examples of this will be used; e.g. what would Envoy put in this? What is the key space for the metadata?

Reply:
Ideally this would contain the same information that we populate in LbEndpoint.metadata.

Thinking of this as a custom response to the Load Balancer for the Endpoint. e.g. if Load Balancer is using some sort of 'affinity', it would pass down it down to envoy via LbEndpoint.metadata. And should be copied back in the report to the balancer, for some correlation from Assignment to Load Report at the balancer.

That said, I agree that this is not an exact definition of how this field will be populated. 
I will remove it.